android/quest/src/main/java/org/smartregister/fhircore/quest/ui/family/removemember/BottomSheetScaffold.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2021 Ona Systems, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.smartregister.fhircore.quest.ui.family.removemember
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.compose.ui.platform.ComposeView
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.smartregister.fhircore.engine.ui.theme.AppTheme
+
+class BottomSheetScaffold(
```

Comment:
`BottomSheetScaffold` should be a composable view. I meant using it instead of the fragment.